javascriptv3/example_code/reactnative/App.js

```diff
@@ -0,0 +1,109 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+
+ABOUT THIS NODE.JS EXAMPLE: This sample is part of the SDK for JavaScript Developer Guide
+topic https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/getting-started-react.html.
+
+Purpose:
+App.js contains code enabling a React Native app to create and delete Amazon Simple Cloud Storage (Amazon S3) buckets.
+
+Inputs:
+- REGION
```

Comment:
Suggest adding examples of canonical values here, especially for those new to AWS and/or Amazon Cognito?